src/Guesser/TypeGuesser.php

```diff
@@ -24,7 +24,7 @@
  */
 class TypeGuesser extends AbstractTypeGuesser
 {
-    public function guessType($class, $property, ModelManagerInterface $modelManager)
+    public function guessType(string $class, string $property, ModelManagerInterface $modelManager): ?TypeGuess
```

Comment:
At first sight, this method seems to not be returning `null`, but `void`.
Maybe we could move the return value from the `default` case of the last `switch` to the end of the method in order to improve readability.
BTW, the ORM implementation for this method is not returning `null` neither, so I think we could check if this is really necessary to use `?` in the return type declaration.

Reply:
In this case it returns always a `TypeGuess`. I agree moving the `default` case, but I'd prefer to do it in `3.x`.

The thing about the return type declaration is that `TypeGuesserInterface::guessType` is also implemented by `FilterTypeGuesser` and in this case it returns null.

https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/blob/237777afdb158886fd4c1b7c8b6d669836510bd9/src/Guesser/FilterTypeGuesser.php#L41-L45